garage/tf/core/layers.py

```diff
@@ -833,8 +850,8 @@ def __init__(self, incoming, shape, **kwargs):
                 raise NotImplementedError
                 # if s.ndim != 0:
```

Comment:
Should here be deleted?

Reply:
According to git blame these comments have been here since 2016-08-07, so they don't seem to be useful anymore and I will remove them.